finesse/hardware/__init__.py

```diff
@@ -5,11 +5,9 @@
 from pubsub import pub
 
 if "--dummy-em27" in sys.argv:
-    from .dummy_em27_scraper import DummyEM27Scraper as EM27Scraper
-    from .opus.dummy import DummyOPUSInterface as OPUSInterface
+    from .plugins.em27.dummy_opus_interface import DummyOPUSInterface as OPUSInterface
```

Comment:
If the user is able (or will be able) to select the devices to use after starting the software, is this check still relevant. Indeed, is it relevant to read arguments from the command line anymore?

Reply:
For now, we still need it, because this is how we choose between the real and dummy OPUS device. Once #363 is implemented we can remove it.

(Confusingly, there are two spectrometer-related "devices" in FINESSE: one to scrape sensor values and one to control recording with the spectrometer.)